db/crud.go

```diff
@@ -187,6 +187,31 @@ func (c *DatabaseCollection) GetDocSyncData(ctx context.Context, docid string) (
 
 }
 
+// This gets *just* the Sync Metadata (_sync field) rather than the entire doc, for efficiency
+// reasons. Unlike GetDocSyncData it does not check for on-demand import; this means it does not
+// need to read the doc body from the bucket.
+func (db *DatabaseCollection) GetDocSyncDataNoImport(ctx context.Context, docid string, level DocumentUnmarshalLevel) (SyncData, error) {
+	if db.UseXattrs() {
+		var err error
+		if key := realDocID(docid); key == "" {
+			err = base.HTTPErrorf(400, "Invalid doc ID")
```

Comment:
```suggestion
			err = base.HTTPErrorf(400, "Invalid doc ID: %q", base.UD(docid))
```

Reply:
Adam pointed out I can remove the test that causes this error.